try:
    n = int(input())

except Exception as e:
    print(e)